src/test/java/org/terasology/launcher/util/TestLauncherDirectoryUtils.java

```diff
@@ -64,56 +64,4 @@ public void testGameDirectory() throws IOException {
         Files.createFile(saveFile);
         assertTrue(containsGameData(gameDirectory));
     }
-
-    @Test
-    public void testApplicationDirectoryWindows() {
-        Path expectedApplicationPath = Paths.get("C:/Users/Test/AppData/Roaming/Unit Test");
-        PowerMockito.mockStatic(System.class);
-
-        when(System.getenv("APPDATA")).thenReturn("C:/Users/Test/AppData/Roaming");
-        when(System.getProperty("user.home", ".")).thenReturn("C:/Users/Test");
-
-        assertEquals(expectedApplicationPath, getApplicationDirectory(OperatingSystem.WINDOWS_8, "Unit Test"));
```

Comment:
In order to keep these test we would need a way to create `Platform` instances, something like `new Platform("windows", "64")` or `new Platform("Windows 10", "x86_64")`. 

That, on the other hand, leads to the question whether we would actually keep the enum of sorts. In the end, we only care about these platforms. (this also means that my head did the whole circle now, coming from `OperatingSystem` as it was before... 😕  )

```java
public enum Platform {
	WINDOWS, LINUX, MAC, OTHER;
}
```